content/departments/product-engineering/engineering/cloud/delivery/on-call.md

```diff
@@ -0,0 +1,126 @@
+The manager of the Delivery team exists as the terminal escalation for the on-call rotation.
+
+---
+
+# Delivery Team On-Call Playbook
+
+## Starting On-Call Rotation
+
+At the start of their shift, the Incoming Engineer On-Call (IEOC) will prepare to take over on-call duties from the Outgoing Engineer On-Call.
+
+### Review Hand-off Notes
+
+The previous on-call engineer should post in the #delivery channel a link to the [Delivery On-Call Hand-off Google Doc](https://docs.google.com/document/d/1K1ZsfocwLk9F6Do4DZQImiSmh-ln2L3IomDBlxxdQHs/edit?usp=sharing)
+
+This document is designed to help the Incoming Engineer On-Call get up to speed with any open issues that require their attention during their shift. Additionally, this is designed to provide a mechanism for the Outgoing Engineer On-Call to
+offload support duties in order to transition back into project work.
+
+### Ensure OpsGenie can page you
+
+This can be done in the OpsGenie interface by supplying your cellphone number or by downloading the Opsgenie app.
+The method you choose to be contacted is up to you as long as your response time follows the [Reponse Time guidelines](#response-times-and-slas)
+
+### Join Slack Channels
+
+The following Slack channels should be monitored during your shift. There is no expectation to actively monitor these channels 24/7 during your support week, but you should serve as the designated point of contact during your working hours.
+
+| Channel Name              | Description                                                                                                    |
+| ------------------------- | -------------------------------------------------------------------------------------------------------------- |
+| #delivery                 | Public channel where Soucegraph teammates can ask questions, provide information, and escalate support issues. |
+| #alerts-managed-instances | Application-level webhook alerts from Managed Instances.                                                       |
+| #incidents                | Company-wide incident feed.                                                                                    |
+| #prod-eng-announcements   | Product Engineerng Announcements.                                                                              |
+| #ask-prod-eng             | Company-wide forum to discuss Product Engineering topics. Generally questions are tagged with "[Delivery]"     |
+| #buildkite-main           | Information about BuildKite CI.                                                                                |
+
+|
+
+#### The `@delivery-support` Slack handle
+
+You will be automatically added to the `@delivery-support` Slack group by Opsgenie. This handle is used to call attention to
+important topics in Slack that may or may not include incidents or P1 customer support issues.
+
+While an immediate response is not necessarily required, use your best judgement to determine an appropriate response.
```

Comment:
Being on support doesn't mean you have to answer / resolve all the questions - just provide an initial response, triage, and make sure issues are tracked in Github. On call engineers can ask for help from teammates in answering / troubleshooting and from the manager for prioritizing.